### ISSUE TITLE
Things that need to be updated with new design

### ISSUE THREAD

**Issue by GuillaumeGomez** [gtk-rs/gtk-rs.github.io#207]
* [ ] Add a case with "more blog posts here"?
* [ ] Replace italic in text with bold (looks a bit weird currently)
* [ ] We're not part of GNOME Circle yet :)

**Comment by bilelmoussaoui** [comment]
- comment gtk4 from the list of crates for now as it's not released yet
- for me: re-add gtk3 stable docs to gtk3-rs so that the link works there as well

**Comment by sophie-h** [comment]
- Fonts and computers is still hell. How can you mess up italic so badly.
- Circle is fixed

> * comment gtk4 from the list

It's JSON not sure if comment works :p

- Center: :unamused: ^^

**Comment by GuillaumeGomez** [comment]
What's the problem with center? All the content on the page is centered after all. :p

**Comment by sophie-h** [comment]
- [ ] Link `/docs-src/` to start page (since it is linked from other pages)

**Comment by Hofer-Julian** [comment]
- [ ] Remove all usage of italic
- [ ] Use monospace for gtk-rs

**Comment by Hofer-Julian** [comment]
One thing I've noticed is that people who have no idea about gtk, glib or GNOME will probably be quite confused what gtk-rs is about. How about adding a nice screenshot prominently at the top?

**Comment by GuillaumeGomez** [comment]
We had one, let's put it back!

**Comment by sophie-h** [comment]
> We had one, let's put it back!

Julian said **nice**

**Comment by GuillaumeGomez** [comment]
You hurt my feelings. :crying_cat_face:

**Comment by bilelmoussaoui** [comment]
in smaller sizes, the menu looks like this. it could be made a bit nicer :p 
![image](https://user-images.githubusercontent.com/7660997/121089243-28c05c00-c7e7-11eb-8477-b7e254b183f8.png)

**Comment by Hofer-Julian** [comment]
> You hurt my feelings. 😿

Sorry @GuillaumeGomez, but I have to agree with Sophie 😊

Hard to choose which one though.
Shortwave? 
It is quite pretty and it is neutral in the sense that no one from us was involved with it.

**Comment by Hofer-Julian** [comment]
-[ ] Remove apps without development in the past 12 months

**Comment by sophie-h** [comment]
>  Remove apps without development in the past 12 months

Nothing that "need to be updated with new design" imo

**Comment by Hofer-Julian** [comment]
Fair enough

**Comment by bilelmoussaoui** [comment]
- [ ] move the RSS feed to the footer along with links?

**Comment by GuillaumeGomez** [comment]
@bilelmoussaoui We already have a big footer, so maybe not? Or maybe change the display of the footer?

**Comment by sophie-h** [comment]
> You say "go look for the projects link yourself" whereas we could simply generate a link to an anchor close to it.

But the table is directly below the text. Not sure if a link helps.

**Comment by GuillaumeGomez** [comment]
Oh I see, maybe reword it a bit to say it's just below?

**Comment by sophie-h** [comment]
I don't know why but https://gtk-rs.org/docs-src/ gives me a 404 and locally I get the redirection as intended. Maybe CDN?

The second 'regression' that is not fixed is the missing screenshot. I would suggest that we fix that via a curated list of 'projects using gtk-rs' where we have a slideshow of screenshots.

For everything else I would tend to having new issues and I wouldn't guarantee that I will implement them.

**Comment by sophie-h** [comment]
> * move the RSS feed to the footer along with links?

What the page has certainly be lacking is the possibility to just throw gtk-rs.org into a feed reader. I will try to add the corresponding meta data.

**Comment by GuillaumeGomez** [comment]
That would make the page very heavy, not sure if this would be a good idea...

**Comment by GuillaumeGomez** [comment]
We already provide a RSS link, but maybe it's not as easy to find as it should?

**Comment by sophie-h** [comment]
> That would make the page very heavy, not sure if this would be a good idea...

Heavy in what sense? I had something like this in mind. The required space would about the same.

![IMG_2058](https://user-images.githubusercontent.com/3466497/121155619-4245c000-c848-11eb-927b-f83800fa1abe.jpg)

**Comment by GuillaumeGomez** [comment]
Heavy not in design but in data. :)

**Comment by sdroege** [comment]
> We already provide a RSS link, but maybe it's not as easy to find as it should?

I think we're talking about `<link rel="alternate" type="application/rss+xml" ...>` here or not?

**Comment by sophie-h** [comment]
The `/docs-src/` link works! I think there is a lot that can be checked in the issue.

Thinks I will do later today

- [ ] Add feed link to `<head>`
- [ ] Fix wording for "Projects"
- [ ] Make drop down menu a bit nicer

I think for the rest we might want to have new issues.

**Comment by sophie-h** [comment]
> Heavy not in design but in data. :)

I'm thinking about 3 to 4 apps with screenshots (also since they need some maintenance) and keeping the list of all apps somewhat hidden under a link/dropdown (idk yet) that shows off with 'And 65 other apps.'

**Comment by GuillaumeGomez** [comment]
Ok, that seems like a good idea!

**Comment by sophie-h** [comment]
Replace italic in text with bold (looks a bit weird currently)
 We're not part of GNOME Circle yet :) (fixed in #206)
 Comment gtk4 from the list of crates for now as it's not released yet
 Center "If you want your app to be added to this list, please create a Pull Request for it."?
 Link /docs-src/ to start page (since it is linked from other pages)
 Put back application screenshot somewhere on the index page
 Add feed link to <head>
 Fix wording for "Projects"
 Make drop down menu a bit nicer

> * [ ]  Add a case with "more blog posts here"?
> * [x]  Replace italic in text with bold (looks a bit weird currently)
> * [x]  We're not part of GNOME Circle yet :) (fixed in #206)
> * [x]  Comment gtk4 from the list of crates for now as it's not released yet
> * [ ]  Center "If you want your app to be added to this list, please create a Pull Request for it."?

It's consistent with the rest of the default texts. Let's maybe skip the discussion because #213 will redo this area anyways?

> * [ x  Link `/docs-src/` to start page (since it is linked from other pages)
> * [ ]  Put back application screenshot somewhere on the index page

Is #213 enough to replace this?

> * [x]   Add feed link to `<head>`
> * [x]  Fix wording for "Projects"
> * [x]  Make drop down menu a bit nicer
> * [ ]  Improve wording (for example, "provides safe Rust bindings for fundamental libraries" instead of "provides safe bindings to the Rust language")
> * [ ]  Projects using gtk-rs: Add carousel/slideshow (#213)

**Comment by GuillaumeGomez** [comment]
I ticked the boxes. :)

**Comment by sophie-h** [comment]
We currently have this in some docs

![image](https://user-images.githubusercontent.com/3466497/121776475-5ca3d480-cb8d-11eb-8706-9f38b1d0d771.png)

First links to `docs-src` second currently links to glib git docs.

Maybe replace with

- gtk-rs project overview → gtk-rs.org
- gtk-rs core documentation overview (including GLib and GObject binidings) → gtk-rs.org/gtk-rs-core

Not sure.

**Comment by sophie-h** [comment]
Wait, we can directly link to the correct version of glib docs using `@crate::glib` or what ever!

**Comment by bilelmoussaoui** [comment]
You can also use `mod@glib`?

**Comment by sophie-h** [comment]
> You can also use `mod@glib`?

That's what I meant with "what ever" :) (forgot the syntax)

**Comment by sophie-h** [comment]
Ah, nice. Looks like past me already did that :) Sorry for the spam
